Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -227,6 +230,17 @@ public int GatewayModeMaxConnectionLimit
         /// <seealso cref="CosmosClientBuilder.WithRequestTimeout(TimeSpan)"/>
         public TimeSpan RequestTimeout { get; set; }
 
+        /// <summary>
+        /// Gets the pooled connection timeout in minutes when connected to the Azure Cosmos DB service.
+        /// This property defines maximal connection lifetime in the pool, tracking its age from when the connection was established, 
+        /// regardless of how much time it spent idle or active. Connections are not torn down while actively being used to service requests. 
+        /// This lifetime is useful in order to allow connections to be reestablished periodically so as to better reflect DNS or other network changes.
+        /// This value is only used in .NET version 6.0 and above.
+        /// </summary>
+        /// <value>Default value is 5 minutes.</value>
+        /// <seealso href="https://learn.microsoft.com/en-us/dotnet/api/system.net.http.socketshttphandler.pooledconnectionlifetime?view=net-8.0"/>       
+        public TimeSpan PooledConnectionLifetime { get; set; }
```

Comment:
We need to be explicit about which protocol it affect

```suggestion
        public TimeSpan HttpPooledConnectionLifetime { get; set; }
```